ck_pr.h causes stringop-overread warning with GCC 11

Witnessed when building a library that consumes libck on Ubuntu 21.10:
```
In file included from /opt/circonus/include/ck_pr.h:52,
                 from /opt/circonus/include/mtev_hooks.h:45,
                 from ./util/libsnowth_hooks.h:4,
                 from util/libsnowth_hooks.c:1:
/opt/circonus/include/gcc/x86_64/ck_pr.h: In function ‘ck_pr_md_load_char_16’:
/opt/circonus/include/gcc/x86_64/ck_pr.h:228:17: error: ‘ck_pr_load_64_2’ reading 16 bytes from a region of size 2 [-Werror=stringop-overread]
  228 |                 ck_pr_load_64_2((const uint64_t *)(const void *)t,              \
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  229 |                                 (uint64_t *)(void *)v);                         \
      |                                 ~~~~~~~~~~~~~~~~~~~~~~
```

Seen as recently as `2265c78` on master.

`gcc (Ubuntu 11.2.0-7ubuntu2) 11.2.0`

--- a/include/gcc/x86_64/ck_pr.h
+++ b/include/gcc/x86_64/ck_pr.h
@@ -223,7 +223,7 @@ ck_pr_load_ptr_2(const void *t, void *v)
 
 #define CK_PR_LOAD_2(S, W, T)							\
 	CK_CC_INLINE static void						\
-	ck_pr_md_load_##S##_##W(const T t[2], T v[2])				\
+	ck_pr_md_load_##S##_##W(const T t[W*sizeof(T)], T v[W*sizeof(T)]) 	\
 	{									\
 		ck_pr_load_64_2((const uint64_t *)(const void *)t,		\
 				(uint64_t *)(void *)v);				\

--- a/include/gcc/x86_64/ck_pr.h
+++ b/include/gcc/x86_64/ck_pr.h
@@ -223,7 +223,7 @@ ck_pr_load_ptr_2(const void *t, void *v)
 
 #define CK_PR_LOAD_2(S, W, T)							\
 	CK_CC_INLINE static void						\
-	ck_pr_md_load_##S##_##W(const T t[W*sizeof(T)], T v[W*sizeof(T)]) 	\
+	ck_pr_md_load_##S##_##W(const T t[W], T v[W]) 				\
 	{									\
 		ck_pr_load_64_2((const uint64_t *)(const void *)t,		\
 				(uint64_t *)(void *)v);				\

